### DISCUSSION TITLE
Creating nested numbering

### DISCUSSION THREAD

**Discussion by mwashambw** [dolanmiu/docx#2384] [Q&A]
I want to create a word document that look like the following, How can I can I archive this? 

1. (a)  Some text here
    (b) Another text here

2. (a) Some text
    (b) Some text here

**Comment by dolanmiu** [comment]
Yes, this is possible, check out this demo here:

https://github.com/dolanmiu/docx/blob/master/demo/64-complex-numbering-text.ts
